plugins/PrivacyManager/PrivacyManager.php

```diff
@@ -72,6 +73,8 @@ class PrivacyManager extends Plugin
     private $dntChecker = null;
     private $ipAnonymizer = null;
 
+    private $logger = null;
```

Comment:
Looks unused 🤔  Was there anything planned for it?

Reply:
Yep, a leftover from the tests debugging kerfuffle.